src/Form/SettingsForm.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\geocoder\Form\SettingsForm.
+ */
+
+namespace Drupal\geocoder\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * The geocoder settings form.
+ */
+class SettingsForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'geocoder_settings';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('geocoder.settings');
+
+    $form['cache'] = [
+      '#type' => 'checkbox',
+      '#title' => 'Should we cache the results ?',
```

Comment:
One more minor. The should not be a space before "?". Also we need to make this strung translatable, right? So $this->t().